app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusEvent.kt

```diff
@@ -1,11 +1,14 @@
 package org.simple.clinic.summary.teleconsultation.status
 
 import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus
+import org.simple.clinic.widgets.UiEvent
 
-sealed class TeleconsultStatusEvent
+sealed class TeleconsultStatusEvent : UiEvent
 
 data class TeleconsultStatusChanged(val teleconsultStatus: TeleconsultStatus) : TeleconsultStatusEvent()
```

Comment:
Should we add an analytics name for this as well since it's triggered via a UI event?

Reply:
My only concern was what if the user keep changes that radio button in the same session we will have few analytics events for that. If that's fine I can implement it.

Reply:
I'd say that was the desired behaviour since it's user analytics.